src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -63,8 +71,14 @@
           + "    LOCATION = '<destinationParquetFile>',\n"
           + "    DATA_SOURCE = [<destinationDataSourceName>],\n" // metadata container
           + "    FILE_FORMAT = [<fileFormat>]\n"
-          + ") AS SELECT * FROM\n"
-          + "    OPENROWSET(\n"
+          + ") AS SELECT datarepo_row_id,<columns:{c|"
+          + "          <if(c.isFileType)>"
+          + "             'drs://<hostname>/v1_<snapshotId>_' + [<c.name>] AS [<c.name>]"
```

Comment:
I disagree.  This is doing it on the fly and this is the latest place we can put it (since this is what generates the object that is then shared with users directly in azure).  This also is what we do in BQ

Reply:
OK, I understand. The URLs must be stored as DRS since users access BQ or parquet data directly. The snapshot table in parquet (and view in BQ) are created entirely in SQL so it would be awkward to round trip the data to java just to do the DRS construction there.